src/entry/index.js

```diff
@@ -1,5 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
 import App from './App';
+import reducer from '../reducers/reducer';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const store = createStore(reducer);
```

Comment:
We need to see the redux store in dev tools while developing, so when you create a store, would you add dev tools?